consensus/state.go

```diff
@@ -1386,9 +1385,7 @@ func (cs *ConsensusState) recordMetrics(height int64, block *types.Block) {
 		)
 	}
 
-	cs.metrics.NumTxs.Set(float64(block.NumTxs))
 	cs.metrics.BlockSizeBytes.Set(float64(block.Size()))
-	cs.metrics.TotalTxs.Set(float64(block.TotalTxs))
```

Comment:
Why remove the metric?

Reply:
same as https://github.com/tendermint/tendermint/pull/3783/files#r302011315